src/Count.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of GitHub Profile Views Counter.
+ *
+ * (c) Anton Komarev <anton@komarev.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Komarev\GitHubProfileViewsCounter;
+
+use Webmozart\Assert\Assert;
+
+final class Count
+{
+    /**
+     * Wrong behavior around extremely large values if set
+     * to an extremely large value (near PHP_INT_MAX)
+     * Shouldn't exceed PHP_INT_MAX to avoid casting to a float
+     */
+    private const MAX_COUNT = 1E15;
```

Comment:
can we use PHP_INT_MAX ?

Reply:
The problem with `PHP_INT_MAX` and values near it is that comparison against some floats leads to incorrect results. Like this example:
```php
<?php
$countSum = 123 + PHP_INT_MAX;
echo ($countSum <= PHP_INT_MAX) . "\n"; // 1 (wrong)
```

Reply:
This probably has to do with how floats are stored and how PHP compares a float to an int. Using `<` instead of `<=` fixes it.